server/package.json

```diff
@@ -20,7 +20,7 @@
     "@sentry/node": "^6.2.5",
     "@sentry/tracing": "^6.2.5",
     "axios": "^1.6.0",
-    "brave-alert-lib": "github:bravetechnologycoop/brave-alert-lib#v11.0.0",
+    "brave-alert-lib": "github:bravetechnologycoop/brave-alert-lib#mdb-sensor-reset-prompt",
```

Comment:
Reminder to update this version once the `brave-alert-lib` side is done and deployed: https://github.com/bravetechnologycoop/brave-alert-lib/pull/66